### ISSUE TITLE
Allow scrolling instead of paging

### ISSUE THREAD

**Issue by JakeHuneau** [junegunn/goyo.vim#211]
Is there any way for my text to scroll instead of switch between pages as I am moving vertically?

**Comment by hcf-n** [comment]
Anyone figured out this one? I'm having the same question.

**Comment by samyak039** [comment]
scrolling is working fine for me, can you please elaborate..??

**Comment by hcf-n** [comment]
I use softwrap. With softwrap scrolling is not by visual line.

**Comment by samyak039** [comment]
I have set this up in my NeoVim config, this make the vertical scrolling by the visible lines instead of actual lines, this is helpful in some cases when a single line is too long.
```
nnoremap j gj
nnoremap k gk
```

but befor adding this also scrolling was working fine,

**Comment by hcf-n** [comment]
Yes, but at the bottom of the screen, vim will still jump to the next actual line.

**Comment by samyak039** [comment]
isn't it what it is supposed to be..?

**Comment by hcf-n** [comment]
Yes, this seems to be a feature 
https://vi.stackexchange.com/questions/4507/scrolling-issues-when-the-wrap-option-is-set
